src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx

```diff
@@ -442,16 +478,16 @@ export default function DownloadDialog({
           <div className="InstallModal__size">
             <FontAwesomeIcon
               className={classNames('InstallModal__sizeIcon', {
-                'fa-spin-pulse': !downloadSize()
+                'fa-spin-pulse': !downloadSize
               })}
-              icon={downloadSize() ? faDownload : faSpinner}
+              icon={downloadSize ? faDownload : faSpinner}
             />
-            {downloadSize() ? (
+            {downloadSize ? (
```

Comment:
I think this is causing a `0` to show up after the loading icon

<img width="119" alt="Screenshot 2023-09-30 at 3 03 06 PM" src="https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/188464/44a1c445-b4ab-4cbb-a100-3857e5d4d428">

Maybe do `!!downloadSize ? (...` to make it a boolean